rinja_parser/src/node.rs

```diff
@@ -1161,6 +1161,31 @@ impl<'a> Let<'a> {
             ),
         );
         let (pws, _, (var, val, nws)) = p.parse_next(i)?;
+        if val.is_none() {
+            let kind = match &var {
+                Target::Name(_) => None,
+                Target::Tuple(..) => Some("a tuple"),
+                Target::Array(..) => Some("an array"),
+                Target::Struct(..) => Some("a struct"),
+                Target::NumLit(..)
+                | Target::StrLit(..)
+                | Target::CharLit(..)
+                | Target::BoolLit(..) => Some("a literal"),
+                Target::Path(..) => Some("a path or enum variant"),
+                Target::OrChain(..) | Target::Placeholder(..) | Target::Rest(..) => {
+                    Some("a pattern")
+                }
+            };
+            if let Some(kind) = kind {
+                return Err(winnow::error::ErrMode::Cut(ErrorContext::new(
+                    format!(
+                        "when you forward-define a variable, you cannot use {kind} inplace of \
```

Comment:
```suggestion
                        "when you forward-define a variable, you cannot use {kind} in place of \
```

Reply:
The dictionary says you're right: <https://www.merriam-webster.com/dictionary/place#phrases-anchor>. Thank you for proofreading! :)